global.json

```diff
@@ -1,17 +1,17 @@
 {
   "tools": {
-    "dotnet": "5.0.100-preview.5.20251.2",
+    "dotnet": "5.0.100-preview.6.20266.3",
```

Comment:
I'll hold off on merging this change as we have seen in dotnet/runtime issues with consuming a new preview 6 SDK: https://github.com/dotnet/runtime/issues/37333#issuecomment-638920779

So I'll block this PR until we figure that out.